src/Modules/Station/Components/StationBrowserItem/StationBrowserItem.tsx

```diff
@@ -0,0 +1 @@
+export { StationBrowserItemInfo } from './StationBrowserItemInfo';
```

Comment:
Why don't you write code here instead of export from another file in the same folder?